webapp/app/templates/components.html

```diff
@@ -103,27 +103,28 @@
     {%- set label = field.label -%}
     {%- set help = field.render_kw['help'] -%}
     {%- set detail = field.render_kw['detail'] -%}
+    {%- set example_input = field.render_kw['example_input'] -%}
 
     {% if field.type == 'RadioField' or field.type == 'UnlockCodeField' or field.type == 'IdNrField' or field.type == 'SteuerlotseDateField' %}
         {% if label.text %} {# Only show legend if there is a legend text to show #}
             <span>
-                <legend class="field-label {{ class }}">{{ label.text }}
+                <legend class="field-label {% if example_input %}field-label-example {% endif %} {{ class }}">{{ label.text }}
```

Comment:
Couldn't we use else for the field-label. Imo it is a little strange that we overwrite the styling of field label with field-label-example instead of exchanging them?

Reply:
As this is going to change a lot with the new frontend, I am not so sure if we might want to change between two different legends (one with and one without example_input). But I think for now that solution is good.

Reply:
Excellent question, I thought you might ask that^^ To me, so far, the only difference between a normal label and a label with example is the spacing. Therefore I decided for a new class instead of replacing the class. Because I imagine, if I want to change something for all field-labels I definitely won't remember also changing it for field-label-example. But, I am with you, we might change it in the future anyways.

Reply:
👍I agree with you. I still think we should `field-label` only in the else case though.